home-work-app/src/app/interfaces/IApiResponse.interface.d.ts

```diff
@@ -0,0 +1,23 @@
+export namespace IApiResponse {
```

Comment:
Why interfaces and types have `.d.ts` extensions?
`.d.ts` files is used for other purposes
types should be declared in ordinary `.ts` files

Reply:
chanded